### PR TITLE
Add Insufficient resource quota error and annotation explanation

### PR DESCRIPTION
**Pull request by LucasSaintarbor**
### Summary

- Referencing [[ENHANCEMENT] If ResourceQuota be used up that add the insufficient resource to vm annotation #3944](https://github.com/harvester/harvester/issues/3944)
- Added explanation for new Insufficient resource quota error and annotation in UI

### Next steps

- Update images 
- Add additional details for explaining the error message and annotation

### CODE REVIEW

File: docs/vm/resource-overcommit.md [hunk 1]
```diff
@@ -58,3 +58,13 @@ To address the issue, Harvester provides an annotation `harvesterhci.io/reserved
...
+
+## Insufficient resource quota
+
+When creating or starting multiple VMs simultaneoysly, the values configured in `overcommit-config` may cause the `ResourceQuota` to be less than the resources used for the batch of VMs. As a result, you will see the following error message:
+
+![insufficient-resource-quota-error](/img/v1.2/vm/insufficient-resource-quota-error.png)
+
+The annotation `harvesterhci.io/insufficient-resource-quota: xxxxx` has been added to the VirtualMachine custom resource to let you know the VM was shutdown due to insufficient resources.
+
+![insufficient-resource-quota-annotation](/img/v1.2/vm/insufficient-resource-quota-annotation.png)
```
Comment (chrisho) [inline docs/vm/resource-overcommit.md:70]: Overcommit doesn't affect ResourceQuota, and here the insufficient annotation has nothing to do with overcommit and should be deleted.
I have created a new PR for the ResourceQuota document that also contains the content here, thanks.

**Comment by vickyhella** [comment]
@LucasSaintarbor Can we safely close this PR as #346 is opened? Thanks!!

**Comment by LucasSaintarbor** [comment]
> @LucasSaintarbor Can we safely close this PR as #346 is opened? Thanks!!

Yes, closing now. Thanks!

**Comment by LucasSaintarbor** [comment]
Closing b/c this request is addressed with https://github.com/harvester/docs/pull/346

**Pull request closed without merging**